packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -81,6 +86,28 @@ class ExpansionPanel {
   /// The widget builder that builds the expansion panels' header.
   final ExpansionPanelHeaderBuilder headerBuilder;
 
+  /// The widget builder that builds the expansion panels' indicator.
+  ///
+  /// If null, [ExpandIcon] will be used as the expansion icon by default.
+  ///
+  /// {@tool sample}
+  ///
+  /// ```dart
+  /// ExpansionPanel(
+  ///  isExpanded: true,
+  ///  headerBuilder: (context, isExpanded) =>
+  ///      Text("This is ExpansionPanel's header"),
+  ///  expansionIndicator: (context, isExpanded) =>
+  ///      Switch(value: isExpanded, onChanged: null,),
+  ///  body: ListTile(
+  ///    title: Text("This is title for ExpansionPanel's body"),
+  ///    subtitle: Text(
+  ///        "This is subtitle for ExpansionPanel's body"),),);
```

Comment:
Prefer single quotes
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-single-quotes-for-strings